frappe/tests/ui_test_helpers.py

```diff
@@ -622,3 +623,28 @@ def add_remove_role(action, user, role):
 		user_doc.remove_roles(role)
 	else:
 		user_doc.add_roles(role)
+
+
+@whitelist_for_tests
+def create_system_manager_user(username):
```

Comment:
No need?

Use this one.  https://github.com/frappe/frappe/blob/2cb492561a2ba790241b081be507b252d8061fac/frappe/tests/ui_test_helpers.py#L429-L450